src/errorhandler.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable max-len */
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const isSearchParamsValid = (textInput, caseInput) => {
+  let error = '';
+
+  if (!textInput) {
+    error = 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+  } else if (!caseInput) {
+    error = '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+  } else if (!cases.includes(caseInput)) {
+    error = `This case is not supported. Available cases: ${cases}`;
+  }
```

Comment:
Let's remove an `else` and an `else if` constructions, and use only `if()`.
You need to create an array of errors and push this array in some cases. 
In this situation, we will have only 1 error, but we may not have both values


```
if (condition) {
    
}

if (condition) {
  
}
```